src/lib.rs

```diff
@@ -102,13 +102,15 @@ pub static mut PARSEC_TPM_DIRECT_SE_DRIVER: psa_drv_se_t = psa_drv_se_t {
 unsafe extern "C" fn p_init(
     _drv_context: *mut psa_drv_se_context_t,
     _persistent_data: *mut ::std::os::raw::c_void,
-    _location: psa_key_location_t,
+    _location: psa_key_lifetime_t,
 ) -> psa_status_t {
     let mut client = (*PARSEC_BASIC_CLIENT).write().expect("lock poisoned");
 
     #[cfg(logging)]
     env_logger::init();
 
+    log::info!("SE Driver initialization");
```

Comment:
Reckon this is `info` or `trace`?

Reply:
I used `info` to mimick the similar `info` in Parsec 🤓 
But I am not sure actually. That might be an event important enough to be an `info`!